packages/flutter_tools/lib/src/base/clock.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:quiver/time.dart';
+
+import 'context.dart';
```

Comment:
Yeah, I'd rather not move away from AppContext until we can move to a proper DI framework, like `package:inject`.  Manual injection via constructor args always devolves into huge bloat of singletons being passed down the entire object graph from `main()` on down.

Not saying that I love `AppContext`, but I don't want the cure to be worse than the disease.